core/src/main/java/com/scalar/db/transaction/consensuscommit/CoordinatorGroupCommitter.java

```diff
@@ -38,7 +38,7 @@ public static boolean isEnabled(ConsensusCommitConfig config) {
   public static class CoordinatorGroupCommitKeyManipulator
       implements KeyManipulator<String, String, String, String, String> {
     private static final int PRIMARY_KEY_SIZE = 24;
-    private static final char DELIMITER = ':';
+    private static final char DELIMITER = '$';
```

Comment:
Also changed the delimiter for Group Commit from `:` to `$` since `:` is disallowed in the Cosmos DB adapter.